monai/utils/module.py

```diff
@@ -202,11 +202,11 @@ def load_submodules(basemod, load_all: bool = True, exclude_pattern: str = "(.*[
             except OptionalImportError:
                 pass  # could not import the optional deps., they are ignored
             except ImportError as e:
-                raise ImportError(
-                    "Multiple versions of MONAI may have been installed,\n"
-                    "please uninstall existing packages (both monai and monai-weekly) and install a version again.\n"
-                    "See also: https://docs.monai.io/en/stable/installation.html\n"
-                ) from e
+                msg = (
+                    "\nMultiple versions of MONAI may have been installed?\n"
```

Comment:
This is better, but I still don't fully see the motivation of adding this error if we have no idea whether it might be this or anything else.